src/Process/LayeredMaterialNodeProcessing.js

```diff
@@ -66,7 +66,7 @@ function initNodeElevationTextureFromParent(node, parent, layer) {
             elevation.max = max;
         }
 
-        node.setTextureElevation(elevation);
+        node.setTextureElevation(layer, elevation);
         node.material.elevationLayersId =
             parent.material.elevationLayersId;
```

Comment:
you removed it...

Reply:
What do you mean?

Reply:
do you still support material.elevationLayersId ? I tought you proposed to remove it.

Reply:
Ah ok. I pushed a commit removing this line.